app/controllers/cars_controller.rb

```diff
@@ -1,6 +1,10 @@
 class CarsController < ApplicationController
   def index
-    @cars = Car.order(:plate).all
+    if params[:search].present?
+      @cars = Car.where("plate LIKE ?", "%#{params[:search]}%").order(:plate)
```

Comment:
`ILIKE` es como el like pero sin distinguir mayúsculas y minúsculas. ¿Te parece mejor?

```suggestion
      @cars = Car.where("plate ILIKE ?", "%#{params[:search]}%").order(:plate)
```

Reply:
Ok, lo modifico y gracias!!